shared/pages/PartialClosure/PartialClosure.js

```diff
@@ -165,6 +169,27 @@ export default class PartialClosure extends Component {
     }, 2000)
 
     SwapApp.shared().services.room.on('new orders', () => this.checkPair(haveCurrency))
+
+    this.setEstimatedFeeValues()
+  }
+
+  setEstimatedFeeValues = async () => {
+    const { estimatedFeeValues } = this.state
```

Comment:
Тут лучше создать новую пременную `newEstimatedFeeValues` и далее записывать в нее

```suggestion
    const { estimatedFeeValues } = this.state
    let newEstimatedFeeValues = estimatedFeeValues
```